src/components/ContentPresenter.js

```diff
@@ -0,0 +1,66 @@
+import React, {Component} from "react";
+import I from "immutable";
+
+
+const smallImage = require("/Users/alibilgi/Desktop/GithubBadge/public/styles/images/small.png");
+
+// Initial response objects
+let repos = [{}];
+let users = {
+  message: ""
+};
+
+class ContentContainer extends Component {
+
+
+  render() {
+
+    if (!I.List.isList(this.props.responseData)) {
+      repos = JSON.parse(this.props.responseData[0]);
+      users = JSON.parse(this.props.responseData[1]);
+    } else {
+      repos = [{}];
+      users = {
+        message: ""
+      };
+    }
+
+    return (
+        <div className="content">
+            <div className="col1">
+                <div className="row_of_col1">
+                    <a href={users.html_url || "https://github.com/404"}>
+                        <img src={users.avatar_url || "https://avatars2.githubusercontent.com/u/5779565?v=3&s=88"} alt="empty" />
```

Comment:
"alt attribute" specifies an alternate text for an image, when the image cannot be displayed isn't it?